README.md

```diff
@@ -41,11 +52,5 @@ Saurabh Nanda(snanda@ncsu.edu)
 ## License
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 
-This project is licensed under the MIT License - see the LICENSE.md file for details
-
-## Acknowledgments
```

Comment:
We can change Acknowledgements section to include where our templates come from, "to tip our hats", so to speak. Deleting it entirely seems like a bad idea.